docs/operations-guide/environment-variables.md

```diff
@@ -0,0 +1,63 @@
+# Environment variables
+
+Settings in Metabase are created using a custom macro, `defsetting`. This macro helps to create a Settings model that will be cached and stored in the Database model, viewed/modified from the Admin Panel, and automagically creates a corresponding environment variable that can be used (by default) to override that setting.
```

Comment:
I don't think we need to discuss the `defsetting` implementation detail in docs. Could we just say something like

> Most settings in Metabase can be stored in the application database and viewed/modified in the Admin Panel, or set via environment variables. The environment variables always take precedence over stored settings.

Reply:
Yeah I like that.